inspire_schemas/utils.py

```diff
@@ -1079,3 +1080,47 @@ def sanitize_html(text):
 
     This strips most of the tags and attributes, only allowing a safe whitelisted subset."""
     return _bleach_cleaner.clean(text)
+
+
+def get_paths(schema):
+    for key, val in schema.items():
+        if isinstance(val, dict):
+            for subkey in get_paths(val):
+                yield "{key}.{subkey}".format(key=key, subkey=subkey)
+        else:
+            yield key
+
+
+def get_references_for_schemas():
```

Comment:
```suggestion
def get_refs_to_schemas():
```
Please also add a docstring. Also memoization would be good, as we'll call this often and it will never change during the run of the app.